fuzzing/__main__.py

```diff
@@ -0,0 +1,3 @@
+if __name__ == "__main__":
```

Comment:
Previously `./fuzz` ran the test just by importing them.
I've converted this to a module to run as either
```py
import fuzzing
fuzzing.run()
```
or
```bash
python -m fuzzing
```

Reply:
Ah yes nice one - tbh the current fuzzer is a bit of a hack so this is defo an improvement.